examples/import-boss/generators/import_restrict.go

```diff
@@ -169,62 +182,104 @@ func removeLastDir(path string) (newPath, removedDir string) {
 	return filepath.Join(filepath.Dir(dir), file), filepath.Base(dir)
 }
 
-// Keep going up a directory until we find an .import-restrictions file.
-func recursiveRead(path string) (*fileFormat, string, error) {
+// isGoModRoot checks if a directory is the root directory for a package
+// by checking for the existence of a 'go.mod' file in that directory.
+func isGoModRoot(path string) bool {
+	goModFile := filepath.Join(filepath.Dir(path), "go.mod")
```

Comment:
nit: this could be inlined, no need for the extra variable. Also may be put "go.mod" into a constant?